cvat-ui/src/components/header/settings-modal/workspace-settings.tsx

```diff
@@ -42,7 +42,7 @@ export default function WorkspaceSettingsComponent(props: Props): JSX.Element {
         onSwitchAutomaticBordering,
     } = props;
 
-    const minAutoSaveInterval = 5;
+    const minAutoSaveInterval = 1;
```

Comment:
It it necessary change or for test only?

Reply:
@bsekachev, what is preventing us to decrease this parameter? I guess we discussed it sometime.